ethereum/test/test_beefy_light_client.js

```diff
@@ -0,0 +1,95 @@
+const BigNumber = web3.BigNumber;
+const {
+  deployBeefyLightClient, signatureSubstrateToEthereum,
+  createMerkleTree, mine, catchRevert
+} = require("./helpers");
+const fixture = require('./fixtures/beefy-fixture-data.json');
+
+require("chai")
+  .use(require("chai-as-promised"))
+  .use(require("chai-bignumber")(BigNumber))
+  .should();
+
+const { expect } = require("chai");
+
+describe("Beefy Light Client", function () {
+  let accounts;
+  let owner;
+  let userOne;
+  let userTwo;
+  let userThree;
+
+  before(async function () {
+    accounts = await web3.eth.getAccounts();
+    owner = accounts[0];
+    userOne = accounts[1];
+    userTwo = accounts[2];
+    userThree = accounts[3];
+
+    this.timeout(10 * 1000)
+
+    this.validatorAddressOne = "0xE04CC55ebEE1cBCE552f250e85c57B70B2E2625b";
+    this.validatorAddressTwo = "0x25451A4de12dcCc2D166922fA938E900fCc4ED24";
```

Comment:
Not important, but it would be cleaner to put those into the fixture file together with the commitment, proof etc...

Reply:
Just noticed that this has been done in https://github.com/Snowfork/polkadot-ethereum/pull/436, nice!